containers/zammad/docker-entrypoint.sh

```diff
@@ -138,14 +140,19 @@ fi
 if [ "$1" = 'zammad-railsserver' ]; then
   test -f /opt/zammad/tmp/pids/server.pid && rm /opt/zammad/tmp/pids/server.pid
 
+  PUMA_OPTS=''
```

Comment:
do we need the if at all?
why do we not just set the default to 1 and always use the argument?